verification/abi.go

```diff
@@ -127,6 +130,25 @@ type CertifyKeyResp[CurveParameter Curve, Digest DigestAlgorithm] struct {
 	Certificate       []byte
 }
 
+type SignFlags uint32
+
+const (
+	IsSymmetric SignFlags = 1 << 31 // TODO: should be 30, a bug in rust side code.
+)
```

Comment:
Can you update the rust code to match? Should just need to change https://github.com/chipsalliance/caliptra-dpe/blob/3d0ee253bee6cf7220da17f5392de6560827c268/dpe/src/commands/sign.rs#L18

Reply:
Yes done now.